packages/ui/src/TreeSearch/index.md

```diff
@@ -16,32 +16,32 @@ nav:
 ## API
 
 | 参数 | 说明 | 类型 | 默认值 | 版本 |
-| :-- | :-- | :-- | :-- | :-- | --- |
+| :-- | :-- | :-- | :-- | :-- |
 | treeNode | 树结构数据 | Node | [] | - |
 | defaultExpandAll | 是否默认展开所有节点 | boolean | false | 0.9.10 |
```

Comment:
- Please remove `0.9.10` version info also. Because it's incorrect.

![image](https://github.com/oceanbase/oceanbase-design/assets/14918822/91dfcde1-c321-4a29-80aa-8257d6d2388d)

Reply:
- Current PR could be merged and welcome to fix it in next PR.